src/components/ContentBlock.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import { PseudoBox, Box, Image, Flex, theme } from '@chakra-ui/core';
```

Comment:
> Warning: 'Image' is defined but never used

Reply:
Ah, thought I got them all. I'll clean this up.